tests/project/_internal/test_libraries.py

```diff
@@ -49,57 +49,57 @@ def test_hydrates_recursive_dependencies(self, mock_clone, fs):
     @patchfs
     @mock.patch("mbed_tools.project._internal.git_utils.checkout", autospec=True)
     @mock.patch("mbed_tools.project._internal.git_utils.init", autospec=True)
-    def test_does_not_perform_checkout_if_no_git_ref_exists(self, mock_init, mock_checkout, mock_clone, fs):
+    def test_does_not_perform_deploy_if_no_git_ref_exists(self, mock_init, mock_deploy, mock_clone, fs):
```

Comment:
I mentioned this in our call but forgot to comment here; all the tests in this file mock out git operations from `git_utils`, I don't think we need to rename them to deploy.